app/ui-react/packages/ui/src/Connection/ConnectionsGridCell.tsx

```diff
@@ -1,12 +1,7 @@
-import { CardGrid } from 'patternfly-react';
 import * as React from 'react';
 
 export class ConnectionsGridCell extends React.Component {
```

Comment:
Maybe this one should just be removed then?

Reply:
If it's ok, I'd like to make this the very first thing I do in a follow-up PR?